src/src/radio.c

```diff
@@ -1416,7 +1416,7 @@ uint8_t otPlatRadioGetCslAccuracy(otInstance *aInstance)
     return otPlatTimeGetXtalAccuracy() / 2;
 }
 
-uint8_t otPlatRadioGetCslClockUncertainty(otInstance *aInstance)
+uint8_t otPlatRadioGetCslUncertainty(otInstance *aInstance)
```

Comment:
Should we remove the weak implemenation in OT?

We can expect any radio platform supporting CSL should provide this?

Reply:
Removing the weak implementation sounds like a good idea.